lib/flat-compat.js

```diff
@@ -225,8 +222,8 @@ class FlatCompat {
         this[cafactory] = new ConfigArrayFactory({
             cwd: baseDirectory,
             resolvePluginsRelativeTo,
-            eslintAllPath: path.resolve(dirname, "../conf/eslint-all.cjs"),
-            eslintRecommendedPath: path.resolve(dirname, "../conf/eslint-recommended.cjs")
+            getEslintAllConfig: () => ({ settings: { "eslint:all": true } }),
+            getEslintRecommendedConfig: () => ({ settings: { "eslint:recommended": true } })
```

Comment:
:+1: 

Can you also delete those two `.cjs` files, as they are no longer used?

Reply:
Done.